cinema/serializers.py

```diff
@@ -31,15 +40,28 @@ class Meta:
 
 
 class MovieSerializer(serializers.ModelSerializer):
+    image = serializers.ImageField(read_only=True)
```

Comment:
Code Efficiency: Good job on using `serializers.ImageField` for the movie image field instead of `serializers.CharField`. This is the correct way to handle image fields in Django.